python/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
Do we need license files in this top level directory, when we have the licenses one level down together with the real content ? 

I also think that this directory should only contain the phoenixdb subdirectory, and we should remove the README.md file. It's redundant, and since it's not in the root directory of a git repo, it's not very useful in the context of github either.

Perhaps we could just flatten the directory structure, and instead of /python/phoenixdb/stuff, we could have just python-phoenixdb/stuff ?

Reply:
I put them in the python/ as a guess of what we would include in the src-release. The (strict) requirement is that our source release should have LICENSE and NOTICE files in the root of the source release artifact(s) (i.e. the root level of the zip or tarball).

The guidance here http://www.apache.org/legal/src-headers.html#notice speaks in terms of the source release. I don't remember for sure if that also applies to the "root of the source code repository". Convention is that most projects have an L&N file at the root of the repo.

> Do we need license files in this top level directory, when we have the licenses one level down together with the real content ?

The L&N for the queryserver release will be different than the one we have for python. Specifically, all of the stuff in NOTICE _should not_ appear in the queryserver's release NOTICE. So, it would be good, IMO, whatever we consider the "root" of phoenixdb have the L&N there.

> I also think that this directory should only contain the phoenixdb subdirectory, and we should remove the README.md file. It's redundant, and since it's not in the root directory of a git repo, it's not very useful in the context of github either.

Agree.

> Perhaps we could just flatten the directory structure, and instead of /python/phoenixdb/stuff, we could have just python-phoenixdb/stuff ?

Agree.